build-support/bin/ci.py

```diff
@@ -75,6 +77,13 @@ def create_parser() -> argparse.ArgumentParser:
     default=PythonVersion.py36,
     help="Run Pants with this version (defaults to 3.6)."
   )
+  parser.add_argument(
+    "--remote-execution-enabled", action="store_true",
+    help="Pants will use remote build execution remote where possible (for now, the V2 unit tests). "
+         "If running locally, you must be logged in via the `gcloud` CLI to an account with remote "
```

Comment:
Good idea! I'm going to save this for a dedicated PR because it's going to involve some decisions, namely, who do we decide to give remote access to? Are first time contributors eligible? Do we expire their authorization after some time?

Reply:
I'm not suggesting necessarily expanding the feature... just documenting exactly what you would need to use the feature you've already added here (access to the pantsbuild gcloud account).